packages/models-library/src/models_library/utils/specs_substitution.py

```diff
@@ -55,22 +63,37 @@ def substitutions(self):
         return self._substitutions
 
     def set_substitutions(
-        self, environs: dict[str, SubstitutionValue]
+        self, mappings: dict[str, SubstitutionValue] | None = None
```

Comment:
why add `None`?
What I mean is ... does it make sense to have a default here?
Which use case require a call w/o arguments?
```python
resolver.set_substitutions()
```

Reply:
I have no idea why I've added a default here. Looking at the code. But let me drop it. If it is required I will put it back with a reason.